yarn.lock

```diff
@@ -2,6 +2,14 @@
 # yarn lockfile v1
 
 
+"@ampproject/remapping@^2.1.0":
```

Comment:
Are all of these changes in this file from adding `jest-svg-transformer`? Seems like a lot of unrelated packages got changed, which IIRC shouldn't happen with `--frozen-lockfile`.